src/Resources/Request/Customer/GetCustomersRequest.php

```diff
@@ -10,6 +10,7 @@
 class GetCustomersRequest extends AbstractRequest
 {
     public function __construct(
+        /** @var string[]|null */
```

Comment:
For higher level analysis with PHPStan it requires that we define what kind of data the arrays will contain. These are non-standard PHP definitions (string[], etc.) so they can't replace the types in code. 

For constructor arguments PHPStan didn't like before the constructor. But besides that, there are only a limited number of places where we need these extra hints for PHPStan so no need to create an entire PHP Docblock with every parameter defined. That creates a lot of boilerplate that isn't necessary since the IDEs can read the normal type data for the rest.

Reply:
ok, looks like that worked. I'll update the blocks.